Resources/Prototypes/_LostParadise/Entities/Clothing/Uniforms/misc_roles.yml

```
@@ -27,3 +27,14 @@
     sprite: _LostParadise/Clothing/Uniforms/Jumpsuit/blueshield_suit.rsi
   - type: Clothing
     sprite: _LostParadise/Clothing/Uniforms/Jumpsuit/blueshield_suit.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: LPPClothingUniformSkirtleneckBlueshield
+  name: blueshield's skirtleneck
+  description: This is the costume of a man of high rank.
+  components:
+  - type: Sprite
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)